src/metadata.rs

```diff
@@ -129,6 +142,18 @@ impl Metadata {
     pub fn extension(&self) -> Option<&str> {
         self.extension.as_deref()
     }
+    pub fn is_show(&self) -> bool {
+        self.season.is_some()
```

Comment:
Or if the episode is `Some`, it's rare but some shows don't mark seasons and just have a single incrementing episode counter.